apps/atomy-q/API/app/Http/Controllers/Api/V1/RfqController.php

```
@@ -675,16 +687,23 @@ public function updateStatus(Request $request, string $id): JsonResponse
         ]);
     }
 
-    public function duplicate(Request $request, string $id): JsonResponse
+    public function duplicate(Request $request, string $id, IdempotencyServiceInterface $idempotency): JsonResponse
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$id'. (undefined)

(UnusedFormalParameter)